vizro-ai/docs/pages/user-guides/run-vizro-ai-dashboard.md

```diff
@@ -0,0 +1,154 @@
+
+## Retrieve the Python code of the dashboard
+
+To illustrate the process, lets use the example above.
+<!-- vale off -->
+Like the `VizroAI.plot` method, in order to produce more comprehensive output we need to set `return_elements=True`. `return_elements` is a boolean (by default `False`) which determines the return type of `VizroAI.dashboard`.
+
+- If set to `False` it produces a `Vizro` dashboard object.
+- If set to `True`, it returns a class (a Pydantic model) containing both the dashboard object and the code string used to generate it.
+
+!!! example "View dashboard code"
+
+    === "Code"
+        ```py
+        import vizro.plotly.express as px
+        from vizro_ai import VizroAI
+        from vizro import Vizro
+
+        df = px.data.tips()
+        user_question = """
+        Create a one-page dashboard layout with the following components:
+
+        1. Card:
+           - Position: Left of the page
+           - Size: Takes up 1/4 of the total page width
+           - Content: Display the text "This is Tips dataset"
+
+        1. Table:
+           - Position: Right of the card
+           - Size: Takes up the remaining 3/4 of the page width
+           - Content: Display the Tips dataset
+        """
+        vizro_ai = VizroAI(model="gpt-4o-mini")
+        result = vizro_ai.dashboard([df], user_question, return_elements=True)
+
+        print(result.code)
+        ```
+    === "Result"
+        ```py
+        ######## Module Imports ##########
+        from vizro import Vizro
+        from vizro.managers import data_manager
+        from vizro.models.types import capture
+        import vizro.models as vm
+        from vizro.tables import dash_ag_grid
+
+        ########## Data Imports ##########
+        #####!!! UNCOMMENT BELOW !!!######
+        # data_manager["restaurant_bills"] = ===> Fill in here <===
+
+        ###### Callable definitions ######
+
+
+        ########## Object code ###########
+        dashboard = vm.Dashboard(
+           pages=[
+               vm.Page(
+                   id="Tips Data Visualization",
+                   components=[
+                       vm.Card(
+                           id="tips_card_tips_data_visualization",
+                           type="card",
+                           text="This is Tips dataset",
+                           href="",
+                       ),
+                       vm.AgGrid(
+                           id="tips_table_tips_data_visualization",
+                           figure=dash_ag_grid(data_frame="restaurant_bills"),
+                       ),
+                   ],
+                   title="Tips Data Visualization",
+                   layout=vm.Layout(grid=[[0, 1, 1, 1]]),
+                   controls=[],
+               )
+           ],
+           title="Tips Dataset Overview",
+        )
+        ```
+
+To use the above code, you will still need to add three simple steps.
+1. Import your data into the Python environment.
```

Comment:
These can be bullets since you don't have to execute them in a particular order (you just need to do them all), right?